sap-abap/services/ec2/README.md

```diff
@@ -0,0 +1,95 @@
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `sap-abap` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+- [Allocate an Elastic IP address](zcl_aws1_ec2_actions.clas.abap#L101) (`AllocateAddress`)
+- [Associate an Elastic IP address with an instance](zcl_aws1_ec2_actions.clas.abap#L119) (`AssociateAddress`)
+- [Create a security group](zcl_aws1_ec2_actions.clas.abap#L196) (`CreateSecurityGroup`)
+- [Create a security key pair](zcl_aws1_ec2_actions.clas.abap#L178) (`CreateKeyPair`)
+- [Create and run an instance](zcl_aws1_ec2_actions.clas.abap#L140) (`RunInstances`)
+- [Delete a security group](zcl_aws1_ec2_actions.clas.abap#L236) (`DeleteSecurityGroup`)
+- [Delete a security key pair](zcl_aws1_ec2_actions.clas.abap#L218) (`DeleteKeyPair`)
+- [Describe Availability Zones](zcl_aws1_ec2_actions.clas.abap#L273) (`DescribeAvailabilityZones`)
+- [Describe Regions](zcl_aws1_ec2_actions.clas.abap#L345) (`DescribeRegions`)
+- [Describe instances](zcl_aws1_ec2_actions.clas.abap#L294) (`DescribeInstances`)
+- [Enable monitoring](zcl_aws1_ec2_actions.clas.abap#L386) (`MonitorInstances`)
+- [Get data about a security group](zcl_aws1_ec2_actions.clas.abap#L365) (`DescribeSecurityGroups`)
+- [Get details about Elastic IP addresses](zcl_aws1_ec2_actions.clas.abap#L254) (`DescribeAddresses`)
+- [List security key pairs](zcl_aws1_ec2_actions.clas.abap#L326) (`DescribeKeyPairs`)
+- [Reboot an instance](zcl_aws1_ec2_actions.clas.abap#L426) (`RebootInstances`)
+- [Release an Elastic IP address](zcl_aws1_ec2_actions.clas.abap#L465) (`ReleaseAddress`)
+- [Start an instance](zcl_aws1_ec2_actions.clas.abap#L483) (`StartInstances`)
+- [Stop an instance](zcl_aws1_ec2_actions.clas.abap#L523) (`StopInstances`)
+
+
+<!--custom.examples.start-->
+<!--custom.examples.end-->
+
+## Run the examples
+
+### Instructions
+
+
+<!--custom.instructions.start-->
+<!--custom.instructions.end-->
+
+
```

Comment:
This empty Instructions section looks weird. We should either define boilerplate default instructions or maybe remove the section if there's nothing to say?

Reply:
Agreed. This is in many service readmes. Needs a general solution later.